src/styles/_catalog.scss

```
@@ -14,9 +14,16 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition-duration: 300ms;
+
```

```suggestion
transition: color 300ms, transform 300ms;
```
Потрібно вказувати до який властивостей застовуєш transition, по дефолту стоїть all, казали так не робити

---

Thanks. done !